lib/galaxy/webapps/galaxy/services/history_contents.py

```diff
@@ -1118,6 +1126,13 @@ def __index_legacy(
         ids = legacy_params_dict.get("ids")
         if ids:
             legacy_params_dict["ids"] = self.decode_ids(ids)
+
+        object_store_ids = None
+        sharable = legacy_params.sharable
+        if sharable is not None:
```

Comment:
See comment above, I think this should be part of the `dev` version of the endpoint but I might be wrong.